pom.xml

```diff
@@ -5,13 +5,16 @@
 
     <groupId>hu.blackbelt.karaf.features</groupId>
     <artifactId>jooq</artifactId>
-    <version>3.13.2-SNAPSHOT</version>
+    <version>${revision}</version>
     <name>JOOQ karaf feature project</name>
     <packaging>pom</packaging>
 
     <properties>
-        <karaf-version>4.2.5</karaf-version>
+        <karaf-version>4.3.4</karaf-version>
         <jooq-version>3.13.2</jooq-version>
+        <deployOnly>false</deployOnly>
+        <maven>3.8.3</maven>
+        <revision>1.0.8-SNAPSHOT</revision>
```

Comment:
How this number is calculated? The original version was 3.12.2-SNAPSHOT which have been replaced to revision.